### ISSUE TITLE
平台设计

### ISSUE THREAD

**Issue by yeducom** [Wesly-Yu/AutoZone#5]
平台设计框架
和拓展方向，方便补充一下不

对于，大数据测试平台 有规划不

**Comment by Wesly-Yu** [comment]
据我所知，BAT等大厂也没有针对大数据的测试工具和平台




------------------ 原始邮件 ------------------
发件人: "YangPower"<notifications@github.com>;
发送时间: 2019年6月19日(星期三) 上午9:35
收件人: "Wesly-Yu/AutoZone"<AutoZone@noreply.github.com>;
抄送: "Subscribed"<subscribed@noreply.github.com>;
主题: [Wesly-Yu/AutoZone] 平台设计 (#5)




平台设计框架
 和拓展方向，方便补充一下不
 
对于，大数据测试平台 有规划不

**Comment by yeducom** [comment]
没错 现今是这个阶段 暂时没有；可是如果你有浏览，招聘网站的习惯，你就会发现；3年以上的招聘测试、测试开发，都需要 大数据分析、大数据测试的经验；  
楼主这么好的人，，肯定也希望更多人使用 平台，同时也是站在 平台工具链的风口；

**Comment by Wesly-Yu** [comment]
但是我也不知道怎么做一个大数据的分钟平台，我连风控测试怎么做都不知道，更别说大数据了


发自我的iPhone

------------------ 原始邮件 ------------------
发件人: YangPower <notifications@github.com>
发送时间: 2019年6月19日 10:25
收件人: Wesly-Yu/AutoZone <AutoZone@noreply.github.com>
抄送: Wesly-Yu <1633235633@qq.com>, Comment <comment@noreply.github.com>
主题: 回复：[Wesly-Yu/AutoZone] 平台设计 (#5)




没错 现今是这个阶段 暂时没有；可是如果你有浏览，招聘网站的习惯，你就会发现；3年以上的招聘测试、测试开发，都需要 大数据分析、大数据测试的经验；
 楼主这么好的人，，肯定也希望更多人使用 平台，同时也是站在 平台工具链的风口；

**Comment by Wesly-Yu** [comment]
微信和qq你哪个用的多，， 我们在上面聊，qq-1633235633  微信18382374375


发自我的iPhone

------------------ 原始邮件 ------------------
发件人: YangPower <notifications@github.com>
发送时间: 2019年6月19日 10:25
收件人: Wesly-Yu/AutoZone <AutoZone@noreply.github.com>
抄送: Wesly-Yu <1633235633@qq.com>, Comment <comment@noreply.github.com>
主题: 回复：[Wesly-Yu/AutoZone] 平台设计 (#5)




没错 现今是这个阶段 暂时没有；可是如果你有浏览，招聘网站的习惯，你就会发现；3年以上的招聘测试、测试开发，都需要 大数据分析、大数据测试的经验；
 楼主这么好的人，，肯定也希望更多人使用 平台，同时也是站在 平台工具链的风口；
